samples/template/README.md

```diff
@@ -0,0 +1,20 @@
+# Savant module template
+
+Start your own project with this template.
+
+## Develop with IDE
+
+Copy the template, rename. Open template with your favorite IDE.
```

Comment:
If you copy and rename template, when you try to run.py an error occurs because it cannot find overlay and custom_pyfunc. In module.yml the path is samples.template which does not exist.

Reply:
fixed